src/utils/formatters.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { formatX, formatY, humanizeMeasureName } from './formatters';
+
+describe('Formatters', () => {
+  describe('formatX', () => {
+    it('should format a date correctly.', () => {
+      const firstDayOfYear = new Date();
+      firstDayOfYear.setMonth(0, 1);
+      expect(formatX(firstDayOfYear)).toBe('January 01');
+    });
+  });
+
+  describe('formatY', () => {
+    it('should format a whole number correctly.', () => {
+      expect(formatY(42)).toBe('42');
+    });
+
+    it('should format a decimal number correctly.', () => {
```

Comment:
nit: could you specify what "correct" means here? I.e. round to one decimal place, if that is what it does.